apps/openmw/mwworld/worldimp.cpp

```diff
@@ -1314,8 +1314,6 @@ namespace MWWorld
         if (pos.z() < terrainHeight)
             pos.z() = terrainHeight;
 
-        pos.z() += 20; // place slightly above. will snap down to ground with code below
```

Comment:
This line was first added with commit c41f119ba6d4734a8cea85ee4bbada750fa2d386 "Added new game button". I think that offset needs to be there to fix the player's position in the starting boat. Did you test a new game?

Reply:
> Did you test a new game?

Yes, and did not see any issues.
IIRC, there was some magic for small objects on ground, and this magic caused issues with player position during new game. This line was added to workaround this issue.

Later this magic was removed, so now this line does not affect new game.